docs/v1.md

```diff
@@ -59,6 +60,24 @@
 - has(id: `string`) : `boolean`
 - isEmpty() : `boolean`
 
+## Components
+- servers(): `Servers`
+- channels(): `Channels`
+- messages(): `Messages`
+- operations() : `Operations`
```

Comment:
Do we have agree that we will have that in v3?

Reply:
To have what? This is an API, nothing related with what the spec has or doesn't. 
The same we have `root.components()` we have it here under `components`.